pkg/kopia/command/repository.go

```diff
@@ -0,0 +1,91 @@
+package command
+
+import (
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/kanisterio/kanister/pkg/kopia/command/storage"
+)
+
+const (
+	repositorySubCommand      = "repository"
+	connectSubCommand         = "connect"
+	noCheckForUpdatesFlag     = "--no-check-for-updates"
+	overrideHostnameFlag      = "--override-hostname"
+	overrideUsernameFlag      = "--override-username"
+	pointInTimeConnectionFlag = "--point-in-time"
+)
+
+type RepositoryCommandArgs struct {
+	*CommandArgs
+	LocationSecret  *v1.Secret
+	CredsSecret     *v1.Secret
```

Comment:
We would have to refactor these to be file paths. The secrets will be mounted onto the pods that run these commands.
Let's discuss offline